resources/views/cart/overview.blade.php

```diff
@@ -0,0 +1,27 @@
+@extends('rapidez::layouts.app')
+
+@section('title', __('Cart'))
+
+@section('robots', 'NOINDEX,NOFOLLOW')
+
+@section('content')
+    <cart v-cloak>
+        <div v-if="hasItems" class="container" slot-scope="{ cart, hasItems, changeQty, remove }">
+            <x-rapidez-ct::title class="mb-5 text-ct-primary">
+                @lang('Cart')
+            </x-rapidez-ct::title>
+            <x-rapidez-ct::layout class="[&>div:first-child>div>table]:my-0">
```

Comment:
i've added this into the checkout theme as a PR, this is just for the timebeing. i will monitor the pullrequest, and if it gets merged i'll personally delete this.